src/char/Baicang.py

```
@@ -45,23 +48,33 @@ class Baicang(BaseChar):
-        # BaseChar default, so keep a private timestamp for the ~20s cadence.
-        self.planner_handles_arc = True
-        self._baicang_last_arc_time = float("-inf")
+    @staticmethod
+    def is_abyss_team(chars):
+        from src.combat.team_strategies import is_baicang_abyss_team
+
+        return is_baicang_abyss_team(chars)
+
+    @classmethod
+    def request_abyss_return(cls, context: CombatContext, chars, reason: str) -> None:
+        """目标队辅助完成动作后显式请求白藏回场，即使白藏 E/Q 正在冷却。"""
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)